config.cc

```diff
@@ -6,6 +6,7 @@
 #include "wstring_qt.hh"
 #include <QDir>
 #include <QFile>
+#include <QStandardPaths>
```

Comment:
Is this header present in Qt4? If not, the compilation will fail.

Reply:
Sorry, it was added by autocomplete plugin. Removed, it is already present in #if guard.